sync-data-to-another-database/scripts/loadData.js

```diff
@@ -1,107 +1,109 @@
-const
-  fs = require('fs'),
-  readline = require('readline'),
-  program = require('commander'),
-  {
-    Kuzzle,
-    WebSocket
-  } = require('kuzzle-sdk');
-
-
+const fs = require("fs"),
```

Comment:
Nitpicking: This file is the only one that used double quote instead of single quote. Usually we prefer single quote but at least we need to stay consistent across files